app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+import BaseService from './base';
+
+const TYPE = 'ip-camera';
+
+export default class IpCameraService extends BaseService {
+  constructor(props, config) {
+    super(props, config);
+    Object.seal(this);
+  }
+
+  // Public getter only properties.
```

Comment:
__question:__ Sorry I still struggle to understand why do we need this comments ```Public getter only properties.``` or ```// Public methods.``` - isn't it clear that ```get``` means getter and name like ```[p.method]``` means private method?

On the related note, I agree that we should group and order getters -> public methods -> private methods. So that reader/developer sees public interface first and only scroll further down if he needs details in private methods.